feeds/management/commands/populate_feed_data.py

```diff
@@ -13,16 +13,22 @@ def handle(self, *args, **kwargs):
         for feed in models.Feed.objects.exclude(last_updated__gt=timezone.now() - timedelta(days=7)):
             try:
                 feedpage = feed.get_feedpage()
-            except Exception:
-                self.stderr.write('Unable to parse {}'.format(feed.rss_url))
+            except Exception as e:
+                self.stderr.write('Unable to parse {}.\n\t{}'.format(feed.rss_url, e))
                 continue
             feed.title = feedpage.title
             feed.description = feedpage.description
             feed.link = feedpage.link
+            feed.image = feedpage.image
+            feed.cloud = feedpage.cloud
             feed.last_updated = timezone.now()
 
             for category in feedpage.categories:
                 matches = models.Keyword.objects.filter(word=category.text.lower())
-                [feed.tags.add(kw.tag) for kw in matches]
+                for kw in matches:
+                    try:
+                        feed.tags.add(kw.tag)
+                    except Exception:
```

Comment:
Wrong. I'm not sure which exception you're trying to ignore, but I'm sure it's not all of them.